app/models/concerns/authorizable.rb

```diff
@@ -16,7 +16,7 @@ module Authorizable
       if user.nil?
         self.where('1=0')
       elsif user.admin?
-        self.scoped
+        self.where(nil)
```

Comment:
`where(nil)` should return a relation, in the console `Katello::System.deletable.where(:uuid => ['3','2']).where('uuid not in (?)', ['33232'])` passes, but I'll check what the problem is once Jenkins is back.